### DISCUSSION TITLE
Stops suggestions after a couple of hours

### DISCUSSION THREAD

**Discussion by deanmarris** [community/community#8114] [Copilot Conversations]
![git_copilot](https://user-images.githubusercontent.com/61342107/143478705-85b8bf2f-9da7-48ef-b0f4-7389b0c86acb.JPG)
I notice in the last couple of days Copilot stops suggestions on # comments, would this be just an over capacity issue?
I stopped using it last night as nothing was coming up. This morning it was working as expected. About 2 hours in, Just now, no suggestions

**Comment by davecheney** [comment]
What does the View -> Output; GitHub copilot window tell you?

**Reply by deanmarris** [reply]
![Githubcopilot_Deanmarris_no choices](https://user-images.githubusercontent.com/61342107/143528561-48ce67ab-8fb9-462e-b8b6-befbe81259fd.PNG)

I absolutely love using the copilot as a data science person so gutted it has been stopping with no choices. Prior itv was incredible what it suggested and very fast. I have a great internet connection as well

**Reply by deanmarris** [reply]
This sort of question or comment it would normally eat for breakfast

**Reply by deanmarris** [reply]
Dave I am a Kiwi in Colorado BTW :-)

**Reply by deanmarris** [reply]
I see when I just use a .py file and execute to the console It seems to predict. When I am in VSC and use a Jupyter note book inside VSC it seems to stop. 
![Github Copilot  pY in console not in a Jupyter notebook](https://user-images.githubusercontent.com/61342107/143530618-07af4df4-0635-4d98-a3ed-922eec4cc7df.JPG)
